pipeline/backend/ssh/ssh.go

```diff
@@ -47,11 +46,17 @@ func (e *ssh) IsAvailable() bool {
 }
 
 func (e *ssh) Load() error {
-	dir, err := ioutil.TempDir("", "woodpecker-local-*")
+	cmd, err := e.client.Command("/bin/env", "mktemp", "-d")
```

Comment:
I tend to prefer having a prefix for directory created by mktemp, ( just use "woodpecker-ssh-XXXXXXX" as a argument for mktemp, per the man pages), as it help to later diagnose if there is a problem (like Cleanup not cleaning the directories)